tests/unit/presenters/admin/indexation-list-item-presenter-test.php

```diff
@@ -3,27 +3,79 @@
 namespace Yoast\WP\SEO\Tests\Unit\Presenters\Admin;
 
 use Brain\Monkey;
+use Mockery;
+use Monkey\Functions\stubs;
+use Yoast\WP\SEO\Helpers\Indexable_Helper;
 use Yoast\WP\SEO\Presenters\Admin\Indexation_List_Item_Presenter;
 use Yoast\WP\SEO\Tests\Unit\TestCase;
 
 /**
  * Class Indexation_List_Item_Presenter_Test.
  *
  * @coversDefaultClass \Yoast\WP\SEO\Presenters\Admin\Indexation_List_Item_Presenter
- *
+ * @covers \Yoast\WP\SEO\Presenters\Admin\Indexation_List_Item_Presenter
  * @group presenters
  */
 class Indexation_List_Item_Presenter_Test extends TestCase {
 
+	/**
+	 * The indexable helper.
+	 *
+	 * @var Mockery\MockInterface|Indexable_Helper The indexable helper.
+	 */
+	protected $indexable_helper;
+
+	/**
+	 * The class under test.
+	 *
+	 * @var Indexation_List_Item_Presenter The Indexation_List_Item_Presenter being tested.
+	 */
+	protected $instance;
+
+	/**
+	 * Provides a working test class.
+	 *
+	 * @param int $count The total unindexed count.
+	 */
+	public function default_arrange( $count ) {
+		$this->instance = new Indexation_List_Item_Presenter( $count, $this->indexable_helper );
+	}
+
+	/**
+	 * Initializes requirements for all tests.
+	 */
+	public function setup() {
+		parent::setUp();
+
+		$this->indexable_helper = \Mockery::Mock( Indexable_Helper::class );
+		$this->default_arrange( 0 );
+
+		Monkey\Functions\expect( 'add_query_arg' )->andReturn( '' );
+		Monkey\Functions\expect( 'get_platform_version' )->andReturn( '5.5' );
```

Comment:
`get_platform_version` isn't an existing function in WordPress. I don't think this line is necessary.  
```suggestion
```